webapp/channels/src/tests/setup_jest.ts

```diff
@@ -26,15 +26,9 @@ module.exports = async () => {
 configure({adapter: new Adapter()});
 
 global.window = Object.create(window);
-Object.defineProperty(window, 'location', {
-    value: {
-        href: 'http://localhost:8065',
-        origin: 'http://localhost:8065',
-        port: '8065',
-        protocol: 'http:',
-        search: '',
-    },
-});
```

Comment:
This is a compatibility fix for the jsdom upgrade that came with Jest 30 / Node 24.